src/goto-symex/partial_order_concurrency.h

```diff
@@ -35,6 +38,10 @@ class partial_order_concurrencyt:public messaget
     AX_PROPAGATION=8
   };
 
+  /// Build identifier for the read/write clock variable
+  /// \param e: either shared read or shared write event
+  /// \param axiom: the clock variable to be used
```

Comment:
I think this parameter indicates the axiom that the resulting clock id is part of. Would that be right @tautschnig?

Reply:
Yes.

Reply:
I agree; should I make it explicit in the description?